core-services/image-mirroring/node-observability-operator/mapping_node_observability_operator_4_11

```diff
@@ -0,0 +1,2 @@
+registry.ci.openshift.org/ocp/4.11:node-observability-agent quay.io/node-observability-operator/node-observability-agent:latest
```

Comment:
You need to contact the admin of `quay.io/node-observability-operator` for the credentials to push.

Reply:
Changed the promotion target from OCP to this operator's dedicated ImageStream.